sdk/servicebus/test-resources.json

```diff
@@ -0,0 +1,177 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "baseName": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "Base name to append to Azure resources."
+      }
+    },
+    "subscriptionId": {
+      "type": "string",
+      "defaultValue": "[subscription().subscriptionId]",
+      "metadata": {
+        "description": "The subscription id to which the application and resources belong."
+      }
+    },
+    "tenantId": {
+      "type": "string",
+      "defaultValue": "[subscription().tenantId]",
+      "metadata": {
+        "description": "The tenant id to which the application and resources belong."
+      }
+    },
+    "testApplicationOid": {
+      "type": "string",
+      "metadata": {
+        "description": "The client object id to grant access to test resources."
+      }
+    },
+    "testApplicationId": {
+      "type": "string",
+      "metadata": {
+        "description": "The application client id used to run tests."
+      }
+    },
+    "testApplicationSecret": {
+      "type": "string",
+      "metadata": {
+        "description": "The application client secret used to run tests."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "The location of the resources. By default, this is the same as the resource group."
+      }
+    }
+  },
+  "variables": {
+    "authRuleResourceId": "[resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', variables('namespaceName'), variables('sharedAccessKeyName'))]",
+    "contributorRoleId": "b24988ac-6180-42a0-ab88-20f7382dd24c",
+    "namespaceName": "[concat('sb-java-', parameters('baseName'))]",
+    "namespaceDomainNameSuffix": "servicebus.windows.net",
+    "queueName": "[concat('queue-', parameters('baseName'))]",
+    "subscriptionName": "[concat('subscription-', parameters('baseName'))]",
+    "serviceBusDataOwnerRoleId": "090c5cfd-751d-490a-894a-3ce6f1109419",
```

Comment:
Do we also need to give Sender and Receiver role ? https://docs.microsoft.com/en-us/azure/service-bus-messaging/authenticate-application#built-in-rbac-roles-for-azure-service-bus

Reply:
I have it in Line 120